### PR TITLE
refactor: count Carpentry skill towards Skill Average

### LINKED ISSUE #1602: 0.14 - Carpentry is now a skill
It gives +hp for each level
Since it's an official skill, it should count towards Skill Avg and weight calculations... problem is it's not in achievements API? or maybe it is now? needs to be checked

https://hypixel.net/threads/skyblock-patch-0-14-shens-auctions-the-hex-rng-meter-and-more.5074247/

### LINKED ISSUE #1649: Missing pets Not showing
Did You Remove The missing pets Feature? Or is it a bug

### PR DESCRIPTION
**Pull request by DuckySoLucky**
## Description 

Closes #1602 
Closes #1649 

## Example
image

### CODE REVIEW

**Comment by DuckySoLucky** [comment]
Oops, branch was merged with #1654
Doesn't really matter but still :/

**Comment by DuckySoLucky** [comment]
Updated Description
image

**Comment by dukio** [comment]
image
This gives NaN on senither skill on a profile with API off: http://localhost:32464/stats/dukioooo/Lemon

My solution would be to just hide weight if API is off, carpentry is a skill now so it must count towards weight... and without that the weight values are just wrong imo

**Review by dukio — CHANGES_REQUESTED** [review]
image
This gives NaN on senither skill on a profile with API off: http://localhost:32464/stats/dukioooo/Lemon

My solution would be to just hide weight if API is off, carpentry is a skill now so it must count towards weight... and without that the weight values are just wrong imo

**Comment by DuckySoLucky** [comment]
Sure, I guess 🤷‍♂️

File: views/stats.ejs [hunk 1]
```diff
@@ -56,42 +56,52 @@ const slayerInfo = {
...
   "sea_emperor",
   "phantom_fisherman",
-  "zombie_miner",
   "great_white_shark",
+  "zombie_miner",
   "yeti",
   "grim_reaper",
+  "thunder",
+  "lord_jawbus", 
 ];
```
Comment (dukio) [inline views/stats.ejs]: @DuckySoLucky I think you accidentally merged some changes from the sea creatures PR here

**Review by metalcupcake5 — APPROVED** [review]
didnt test

File: views/stats.ejs (hunk 1 above)
Comment (dukio) [reply views/stats.ejs]: @DuckySoLucky if you can just revert the changes to this array then this PR can go live 🙏

**Comment by DuckySoLucky** [comment]
@dukio Why waiting label?

**Comment by dukio** [comment]
> @dukio Why waiting label? 

https://github.com/SkyCryptWebsite/SkyCrypt/pull/1655#discussion_r968825549

**Review by dukio — APPROVED** [review]
lgtm tested

**Comment by dukio** [comment]
@SkyCryptWebsite/developers 2 approvals 😄

File: views/stats.ejs (hunk 1 above)
Comment (DuckySoLucky) [reply views/stats.ejs]: > @DuckySoLucky if you can just revert the changes to this array then this PR can go live 🙏

Oops I thought I reverted that as well I guess not. Will update in like 5 hours when I get home

Comment (DuckySoLucky) [reply views/stats.ejs]: > @DuckySoLucky I think you accidentally merged some changes from the sea creatures PR here

Just checked file, I have actually fixed it. You were checking outdated file, it says on top as well

**Review by MartinNemi03 — CHANGES_REQUESTED** [review]
About weight systems, they don't seem to be working on any profiles.

**Comment by DuckySoLucky** [comment]
> About weight systems, they don't seem to be working on any profiles.

Solved, accidentally reverted that as well ;-;

**Review by MartinNemi03 — APPROVED** [review]
lgtm, tested

**Pull request merged**